infullmvp-kotlin-basetest/src/main/java/com/infullmobile/android/infullmvp/basetest/InFullMvpBottomSheetDialogFragmentBaseTest.kt

```diff
@@ -27,6 +30,7 @@ abstract class InFullMvpBottomSheetDialogFragmentBaseTest<T : InFullMvpBottomShe
         testedFragment = fragmentController.create().get()
         substituteModules(testedFragment)
         fragmentController.start().resume().visible()
+        testedFragment.setupDialog(testedFragment.onCreateDialog(Bundle()), 0)
```

Comment:
We may want to provide `abstract val onCreateBundle: Bundle`  that we'd pass for the setup.
May prove useful for fragments that do something about that bundle.